src/builder/commands/pip.rs

```diff
@@ -326,6 +326,12 @@ fn version_req(hash: &mut Digest, fname: &Path) -> Result<(), VersionError> {
                 if chunk.len() == 0 || chunk.starts_with("#") {
                     continue;
                 }
+                if chunk.starts_with("-r ") {
```

Comment:
I don't think this parsing is good enough. You need to parse 4 cases I think:
`-rXXX`, `-r XXX`, `--requirement XXX`, `--requirement=XXX`.